person.rb

```diff
@@ -13,7 +13,7 @@ class Person < Nameable
   # Public: Gets/Sets the String name and Interger age of the user.
   attr_accessor :name, :age
   # Public: Returns the Integer id of the user.
-  attr_reader :id
+  attr_reader :id, :rental
```

Comment:
- [ ] We need both   _Getters_ and _Setters_ for _@rentals_ to be present, therefore kindly ensure to mark _rentals_ with _attr_accessor_, like:
```ruby
  attr_accessor :rentals